packages/cubejs-redshift-driver/src/RedshiftDriver.ts

```diff
@@ -0,0 +1,165 @@
+import { PostgresDriver, PostgresDriverConfiguration } from '@cubejs-backend/postgres-driver';
+import { DownloadTableCSVData, UnloadOptions } from '@cubejs-backend/query-orchestrator';
+import { getEnv } from '@cubejs-backend/shared';
+import crypto from 'crypto';
+import { getSignedUrl } from '@aws-sdk/s3-request-presigner';
+import { S3, GetObjectCommand } from '@aws-sdk/client-s3';
+
+interface RedshiftDriverExportAWS {
+  bucketType: 's3',
+  bucketName: string,
+  keyId: string,
+  secretKey: string,
+  region: string,
+}
+
+export interface RedshiftDriverConfiguration extends PostgresDriverConfiguration {
+  exportBucket?: RedshiftDriverExportAWS;
+}
+
+export class RedshiftDriver extends PostgresDriver<RedshiftDriverConfiguration> {
+  public constructor(options: RedshiftDriverConfiguration = {}) {
+    super(options);
+  }
+
+  protected getInitialConfiguration(): Partial<RedshiftDriverConfiguration> {
+    return {
+      exportBucket: this.getExportBucket(),
+    };
+  }
+
+  protected getExportBucket(): RedshiftDriverExportAWS|undefined {
+    const exportBucket: Partial<RedshiftDriverExportAWS> = {
+      bucketType: getEnv('dbExportBucketType', {
+        supported: ['s3']
+      }),
+      bucketName: getEnv('dbExportBucket'),
+      keyId: getEnv('dbExportBucketAwsKey'),
+      secretKey: getEnv('dbExportBucketAwsSecret'),
+      region: getEnv('dbExportBucketAwsRegion'),
+    };
+
+    if (exportBucket.bucketType) {
+      const supportedBucketTypes = ['s3'];
+
+      if (!supportedBucketTypes.includes(exportBucket.bucketType)) {
+        throw new Error(
+          `Unsupported EXPORT_BUCKET_TYPE, supported: ${supportedBucketTypes.join(',')}`
+        );
+      }
+
+      const emptyKeys = Object.keys(exportBucket)
+        .filter((key: string) => exportBucket[<keyof RedshiftDriverExportAWS>key] === undefined);
+      if (emptyKeys.length) {
+        throw new Error(
+          `Unsupported configuration exportBucket, some configuration keys are empty: ${emptyKeys.join(',')}`
+        );
+      }
+
+      return <RedshiftDriverExportAWS>exportBucket;
+    }
+
+    return undefined;
```

Comment:
`return` returns `undefined` by default

Reply:
There is a problem with that because our eslint configuration doesn't allow that.

https://eslint.org/docs/rules/consistent-return#treatundefinedasunspecified